src/components/NavBar.tsx

```diff
@@ -0,0 +1,191 @@
+  const [anchorElAboutUs, setAnchorElAboutUs] =
+    React.useState<null | HTMLElement>(null);
+
+  const handleMenuOpen = (
+    event: React.MouseEvent<HTMLElement>,
+    setAnchorEl: React.Dispatch<React.SetStateAction<null | HTMLElement>>,
+  ) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleMenuClose = (
+    setAnchorEl: React.Dispatch<React.SetStateAction<null | HTMLElement>>,
+  ) => {
+    setAnchorEl(null);
+  };
+
+  return (
+    <Box sx={{ flexGrow: 1 }}>
+      <AppBar
+        position="static"
+        color="default"
+        sx={{ height: '80px', display: 'flex', justifyContent: 'center' }}
+      >
+        <Container>
+          <Toolbar>
+            <Image src="/logo_white.png" alt="Logo" width={60} height={60} />
+            <Box sx={{ flexGrow: 1, display: 'flex', justifyContent: 'right' }}>
+              <Link href="/" passHref>
+                <Button sx={{ color: 'primary.dark' }}>
+                  <Typography variant="body2">Home</Typography>
+                </Button>
+              </Link>
+              <Button
+                color="inherit"
+                onClick={(event) =>
+                  handleMenuOpen(event, setAnchorElMentorship)
+                }
+                sx={{ color: 'primary.dark' }}
+              >
+                <Typography variant="body2">Mentorship</Typography>
+              </Button>
+              <Menu
+                sx={{ color: 'primary.dark', fontSize: '16px' }}
+                anchorEl={anchorElMentorship}
+                open={Boolean(anchorElMentorship)}
+                onClose={() => handleMenuClose(setAnchorElMentorship)}
+              >
+                <MenuItem
+                  onClick={() => handleMenuClose(setAnchorElMentorship)}
+                >
+                  <Link href="/mentorship" passHref>
+                    <Typography
+                      variant="body2"
+                      sx={{ color: 'primary.dark', textDecoration: 'none' }}
+                    >
+                      Overview
+                    </Typography>
+                  </Link>
+                </MenuItem>
+                <MenuItem
+                  onClick={() => handleMenuClose(setAnchorElMentorship)}
+                  sx={{ color: 'primary.dark', fontSize: '16px' }}
+                >
+                  <Link href="/mentors" passHref>
+                    Mentors
+                  </Link>
+                </MenuItem>
+                <MenuItem
+                  onClick={() => handleMenuClose(setAnchorElMentorship)}
+                  sx={{ color: 'primary.dark', fontSize: '16px' }}
+                >
+                  <Link href="/mentorship" passHref>
+                    Resources
+                  </Link>
+                </MenuItem>
+                <MenuItem
+                  onClick={() => handleMenuClose(setAnchorElMentorship)}
+                  sx={{ color: 'primary.dark', fontSize: '16px' }}
```

Comment:
Should we create variable for font-size and re-use? Same for `colour` or this is just simple example to setup?

Reply:
this component is only for testing (MUI and jest)
once the design is in place, vars like fontSize etc. are specified in the theme.